spec/support/page_objects/teacher_interface/application/active_application.rb

```diff
@@ -0,0 +1,37 @@
+module PageObjects
+  module TeacherInterface
+    class TaskListItem < SitePrism::Section
+      element :link, "a"
+      element :status, ".govuk-tag"
+    end
+
+    class TaskListSection < SitePrism::Section
+      element :heading, "h2"
+      sections :task_list_items, TaskListItem, ".app-task-list__item"
+    end
```

Comment:
As these aren't specific to the teacher application page (we've got a similar definition for assessor applications - https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/74595c0b43f1b0bca80e26d27ee2c6af78e2c550/spec/support/page_objects/assessor_interface/application.rb#L15-L24) I'd suggest refactoring them out in to a common location (`spec/support/page_objects`) and using them in both places.